array/removeAt.test.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import removeAt from "./removeAt.ts";
+
+describe("removeAt", () => {
+  it("should remove the element specified by index", () => {
+    expect(removeAt(3)([1, 2, 3, 4, 5, 6])).toEqual([1, 2, 3, 5, 6]);
+  });
+
+  it("should return source array when given index is out of range", () => {
```

Comment:
Consider simplifying this into:

```suggestion
  it("should do nothing when given an index out of range", () => {
```